tests/test_snapwell_norne.py

```diff
@@ -42,6 +43,49 @@ def test_snapwell_well_ok(self):
             wp_in, wp_snapped = self.check_snapwell_metadata_ok(
                 expected_well_output_fname
             )
+            self.assertEqual(
+                ["x", "y", "z", "LENGTH", "TVD_DIFF", "OLD_TVD", "OWC"],
+                wp_snapped.headers(),
+            )
+            self.check_snapwell_wellpath_ok(wp_in, wp_snapped)
+
+    def test_snapwell_well_ok_with_sat(self):
+        with TestAreaContext("Run_snapwell_valid_arg_exit_ok"):
```

Comment:
Can you rewrite to avoid dependency on `TestAreaContext`? It is a home cooked `c` implementation of test utilities, much better to use some `fixtures`.

Reply:
Can do!